src/xtgeo/grid3d/_roff_parameter.py

```diff
@@ -299,6 +299,6 @@ def should_skip_parameter(tag: str, key: str) -> bool:
             nz=roff["nz"],
             name=roff["name"],
             values=roff["values"],
-            code_names=roff.get("code_names", None),
-            code_values=roff.get("code_values", None),
+            code_names=roff.get("code_names"),
+            code_values=roff.get("code_values"),
```

Comment:
New ruff wanted this

Reply:
`ruff` cares about `roff` 😄 . I think None is default; probably why